Makefile

```diff
@@ -54,7 +54,7 @@ build-docs:
 
 test:
 	make deps
-	docker-compose run --rm ${service_name} pytest ${args} --snapshot-update --no-cov -m "not slow and not airflow and not ray and not integration" tests/
+	docker-compose run --rm ${service_name} pytest ${args} --snapshot-update --no-cov -m "not slow and not airflow and not ray and not dvc and not integration" tests/
```

Comment:
I assume this is because this Makefile target is to update the snapshots, as opposed to actually running the tests and comparing the snapshots. 

I don't think any of the current active devs uses the Makefile anyways, I'm just updating it to keep the same behavior but am not very concerned with what that behavior is as of now.